nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/RHostedFacetImpl.java

```diff
@@ -138,8 +82,8 @@ public Asset upload(final String path, final Payload payload) throws IOException
 
   @TransactionalStoreBlob
   protected Asset doPutArchive(final String path,
-                              final TempBlob archiveContent,
-                              final Payload payload) throws IOException
+                               final TempBlob archiveContent,
```

Comment:
Extra whitespace

Reply:
done in fe40919080a00ea9162d0af9be02c414af338e27

Reply:
Sorry, wrong answer here. It's already formatted properly